templates/errors/session-expired.html

```diff
@@ -1,27 +1,9 @@
 {% extends 'layouts/_base.html' %}
-{% from "components/panel/_macro.njk" import onsPanel %}
-
-{% import 'macros/helpers.html' as helpers %}
 
 {% set page_title = _("Session expired") ~ " - " ~ _("Census 2021") %}
 
 {% block main %}
-  <div class="u-mb-s" data-qa="session-expired-error">
-    <h1 {{helpers.track('visible', 'Timeout', 'Timeout expired')}}>{{ _("Your session has expired") }}</h1>
-  </div>
-
-  {% call onsPanel({
-    "type": "success",
-    "spacious": true,
-    "classes": "u-mb-s"
-  }) %}
-    <p>{{ _("To help protect your information we have signed you out") }}.</p>
-    {% if account_service_url %}
-      <p>{{ _('We have saved your progress and you will need to <a href="%(account_service_link)s">enter your unique code</a> to access the survey again.', account_service_link=account_service_url) }}</p>
-    {% endif %}
-    {% if using_edge %}
-      <p>If you have been timed out unexpectedly on more than one occasion, please try using a different browser. If you’re still having problems completing the census, <a href="https://census.gov.uk/contact-us/">contact us</a>.</p>
-    {% endif %}
-  {% endcall %}
-
+  <h1>{{ _("Your session has expired due to inactivity") }}</h1>
```

Comment:
The page title should match h1. Would be good to get confirmation but that is the general principle.